src/blueprint.js

```diff
@@ -142,7 +142,7 @@ module.exports = {
         }
 
         if (is.not.function(validator)) {
-          output.validationErrors.push(`I don't know how to validate ${schema[key]}`)
+          output.validationErrors.push({ key, err: new Error(`I don't know how to validate ${schema[key]}`) })
```

Comment:
Can we add a default 400 status here?

Reply:
Status code property is not a blueprint error convention but our app error convention. Let's keep this library separate in it's own domain